website/docs/development_guides/dev_hello_world.md

```diff
@@ -192,7 +180,7 @@ Now take the following steps to run chaos:
 2. Install Chaos Mesh:
 
     ```bash
-    helm install helm/chaos-mesh --name=chaos-mesh --namespace=chaos-testing --set chaosDaemon.runtime=containerd --set chaosDaemon.socketPath=/run/containerd/containerd.sock
+    helm install chaos-mesh helm/chaos-mesh --namespace=chaos-testing --set chaosDaemon.runtime=containerd --set chaosDaemon.socketPath=/run/containerd/containerd.sock
```

Comment:
how about providing two commands for helm v3 and helm v2?

Reply:
Sure that would be better. thx!